server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/routes/DownloadRoutes.scala

```diff
@@ -219,7 +219,7 @@ class MessagePartBlobResolver @Inject()(val messageIdFactory: MessageId.Factory,
 class BlobResolvers(blobResolvers: Set[BlobResolver]) {
 
   @Inject
-  def this(blobResolvers: java.util.Set[BlobResolver]) {
+  def this(blobResolvers: java.util.Set[BlobResolver]) = {
```

Comment:
Should we remove the brackets here and in other places?

Reply:
Not sure, this is just a code convention, 
When I run scalafix plugin, it suggested that.